libs/libmilter/Makefile

```diff
@@ -0,0 +1,78 @@
+#
+# Copyright (C) 2017 Val Kulkov <val.kulkov@gmail.com>
+#
+# This is free software, licensed under the MIT License
+#
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=sendmail
+PKG_VERSION:=8.15.2
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME).$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://artfiles.org/sendmail.org/pub/sendmail/ ftp://ftp.cs.berkeley.edu/ucb/sendmail/ http://www.netgull.com/sendmail/ http://sendmail.cybermirror.org/
```

Comment:
Done, please review [the commit](https://github.com/openwrt/packages/pull/4369/commits/4bce6115cfbde375f50ff84c486445cf3dc07d8b#diff-607c0602a87f57834f7ebcd862ee8c5f).